src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -984,6 +984,7 @@ public static void onWorldLoad(@NotNull final World world)
                             if (colonyData != null)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 5 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS134)